R/git.R

```diff
@@ -135,10 +135,10 @@ github_remote_parse <- function(x) {
 
   if (grepl("^https", x)) {
     # https://github.com/hadley/devtools.git
-    re <- "github.com/(.*?)/(.*)\\.git"
+    re <- "github[^/]*/(.*?)/(.*)\\.git"
   } else if (grepl("^git", x)) {
     # git@github.com:hadley/devtools.git
-    re <- "github.com:(.*?)/(.*)\\.git"
+    re <- "github[^:]*:(.*?)/(.*)\\.git"
```

Comment:
If both of these regexes are exactly the same there is no need for these conditionals.